node/tests/st/bgp/test_route_reflector_cluster.py

```diff
@@ -1,132 +0,0 @@
-                                        ip_pass_list=[workload_host1.ip,
-                                                      workload_host2.ip])
-            # Bring down a node
-            rrc.redundancy_groups[0][0].cleanup()
-
-            # Assert that network is still connected
-            self.assert_true(workload_host1.check_can_ping(workload_host2.ip, retries=10))
-            self.assert_ip_connectivity(workload_list=[workload_host1,
-                                                       workload_host2],
-                                        ip_pass_list=[workload_host1.ip,
-                                                      workload_host2.ip])
-
-
-    def _test_route_reflector_cluster(self, backend='bird'):
-        """
-        Run a multi-host test using a cluster of route reflectors and node
-        specific peerings.
-        """
-        with DockerHost('host1',
-                        additional_docker_options=CLUSTER_STORE_DOCKER_OPTIONS,
-                        start_calico=False) as host1, \
-             DockerHost('host2',
-                        additional_docker_options=CLUSTER_STORE_DOCKER_OPTIONS,
-                        start_calico=False) as host2, \
-             DockerHost('host3',
-                        additional_docker_options=CLUSTER_STORE_DOCKER_OPTIONS,
-                        start_calico=False) as host3, \
-             RouteReflectorCluster(2, 2) as rrc:
-
-            # Start both hosts using specific backends.
-            host1.start_calico_node("--backend=%s" % backend)
-            host2.start_calico_node("--backend=%s" % backend)
-            host3.start_calico_node("--backend=%s" % backend)
-
-            # Set the default AS number - as this is used by the RR mesh, and
-            # turn off the node-to-node mesh (do this from any host).
-            update_bgp_config(host1, asNum=64513, nodeMesh=False)
-
-            # Create a workload on each host in the same network.
-            network1 = host1.create_network("subnet1")
-            workload_host1 = host1.create_workload("workload1", network=network1)
-            workload_host2 = host2.create_workload("workload2", network=network1)
-            workload_host3 = host3.create_workload("workload3", network=network1)
-
-            # Allow network to converge (which it won't)
-            self.assert_false(workload_host1.check_can_ping(workload_host2.ip, retries=5))
-            self.assert_true(workload_host1.check_cant_ping(workload_host3.ip))
-            self.assert_true(workload_host2.check_cant_ping(workload_host3.ip))
-
-            # Set distributed peerings between the hosts, each host peering
-            # with a different set of redundant route reflectors.
-            for host in [host1, host2, host3]:
-                for rr in rrc.get_redundancy_group():
-                    create_bgp_peer(host, "node", rr.ip, 64513, metadata={'name': host.name + rr.name.lower()})
```

Comment:
Do we have a replacement for this?  IIRC the e2e test does not use multiple RR cluster IDs.